### DISCUSSION TITLE
Bulk action [Apr 20]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#1908] [Discord Threads]
> 🙋‍♂️ **Steven H**  
## Is there a way I could code a plugin or extension or something to add my own custom menu option?  Specifically I want to create a button that does a bulk edit of selected records.    Or is bulk edits an existing feature I just haven't found?  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/966035641450254406/Screen_Shot_2022-04-19_at_10.51.13_AM.png)

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### Not at the moment,    
####    
#### However, there is a feature request for button I think which is nothing but an action defined on a row. That could be made available on the menu.  
#  
> 💁 **o1lab**  
#### https://github.com/nocodb/nocodb/issues/1041  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/966035642410741792)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
